### PR TITLE
fix: Clear notification when app is killed from recent tasks. 

### PR DESCRIPTION
**Pull request by nur-shuvo**
When app is killed from recent task, notification still persists.
This PR fixes https://github.com/libre-tube/LibreTube/issues/5833

### CODE REVIEW

File: app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        return START_NOT_STICKY
+    }
+
+    override fun onTaskRemoved(rootIntent: Intent?) {
+        nManager?.cancel(PLAYER_NOTIFICATION_ID)
+        stopSelf()
+    }
+
+    companion object {
+        private const val PLAYER_NOTIFICATION_ID = 1
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt:31]: That constant is already defined somewhere else (iirc in NotificationHelper.kt`)

File: app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+import android.app.NotificationManager
+import android.app.Service
+import android.content.Intent
+import android.os.IBinder
+import androidx.core.content.getSystemService
+
+class OnClearFromRecentService : Service() {
+    private var nManager: NotificationManager? = null
+
+    override fun onBind(p0: Intent?): IBinder? {
```
Comment (Bnyro) [inline app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt:12]:
```suggestion
    override fun onBind(p0: Intent?): IBinder? = null
```

File: app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt (hunk 1 above)
Comment (nur-shuvo) [reply app/src/main/java/com/github/libretube/services/OnClearFromRecentService.kt:31]: I used it, but in the remote debug build it cannot resolve it, i don't know why. Will I do it ?

**Comment by nur-shuvo** [comment]
@Bnyro Can you please check?

**Review by Bnyro — APPROVED** [review]
Thanks!

**Pull request merged**